daemon/cluster/convert/swarm.go

```diff
@@ -30,13 +31,23 @@ func SwarmFromGRPC(c swarmapi.Cluster) types.Swarm {
 					AutoLockManagers: c.Spec.EncryptionConfig.AutoLockManagers,
 				},
 			},
+			TLSInfo: types.TLSInfo{
+				TrustRoot: string(c.RootCA.CACert),
+			},
+			RootRotationInProgress: c.RootCA.RootRotation != nil,
 		},
 		JoinTokens: types.JoinTokens{
 			Worker:  c.RootCA.JoinTokens.Worker,
 			Manager: c.RootCA.JoinTokens.Manager,
 		},
 	}
 
+	issuerInfo, _ := ca.IssuerFromAPIRootCA(&c.RootCA)
```

Comment:
Please check the error value. It's not necessary to expose it, but I think we should avoid relying on the contents of `issuerInfo` if the function returns an error.